Microsoft Defender Baseline UI Updates (#1764)
cisagov/ScubaGear: PowerShell/ScubaGear/baselines/defender.md

5. From the **Categories** list, select **Custom**.

6. From the **Regulations** list, select **Custom policy** and then click
   **Next**.

7. Edit the name and description of the policy if desired, then click
   **Next**.

8. Under **Assign admin units**,  ensure **Admin units** is set to **Full directory** by default, then click **Next**.
   

> 9. Under **Choose locations to apply the policy**, set **Status** to **On**

Reviewer: ```suggestion
9. Under **Choose where to apply the policy**, set **Status** to **On**
```

   for at least the Exchange email, OneDrive accounts, SharePoint
   sites, Teams chat and channel messages, and Devices locations, then
   click **Next**.

10. Under **Define policy settings**, select **Create or customize advanced
   DLP rules**, and then click **Next**.

11. Click **Create rule**. Assign the rule an appropriate name and
   description.

12. Click **Add condition**, then **Content contains**.

13. Click **Add**, then **Sensitive info types**.

14. Add information types that protect information sensitive to the agency.
    At a minimum, the agency should protect:

    - Credit card numbers
    - U.S. Individual Taxpayer Identification Numbers (ITIN)
    - U.S. Social Security Numbers (SSN)
    - All agency-defined PII and sensitive information

15. Click **Add**.

16. Under **Actions**, click **Add an action**.

17. Check **Restrict Access or encrypt the content in Microsoft 365
    locations**.

18. Under this action, select **Block Everyone**.

19. Under **User notifications**, turn on **Use notifications to inform your users and help educate them on the proper use of sensitive info**.

> 20. Under **Microsoft 365 files and Microsoft Fabric items**, a section that appears after user notifications are turned on, check the box next to **Notify users in Office 365 service with a policy tip, or email notification**.

Reviewer: Seeing the below settings within a gcc environment. May be slightly different for commercial tenants.

<img width="1744" height="376" alt="image" src="https://github.com/user-attachments/assets/6705023a-b420-4c5b-bcc2-456d470ff71a" />

```suggestion
20. Under **Microsoft 365 services**, a section that appears after user notifications are turned on, check the box next to **Notify users in Office 365 service with a policy tip or email notification**.
```

Reviewer: I tested on GCC High as well and the setting doesn't exist. Maybe we word the implementation steps like the following then, 

20. Under **Microsoft 365 services** if using a GCC environment or under **Microsoft 365 files and Microsoft Fabric items** if using a Commercial environment, a section that appears after user notifications are turned on, check the box next to **Notify users in Office 365 service with a policy tip or email notification**.

> 20. Under **Microsoft 365 services** if using a GCC environment or under **Microsoft 365 files and Microsoft Fabric items** if using a Commercial environment, a section that appears after user notifications are turned on, check the box next to **Notify users in Office 365 service with a policy tip or email notification**.

Reviewer: ```suggestion
20. Under **Microsoft 365 services** if using a GCC environment or under **Microsoft 365 files and Microsoft Fabric items** if using a Commercial environment, a section that appears after user notifications are turned on, check the box next to **Notify users in Office 365 service with a policy tip or email notifications**.
```
